common/src/main/java/com/datastax/oss/simulacron/common/result/AlreadyExistsResult.java

```diff
@@ -15,13 +15,13 @@
  */
 package com.datastax.oss.simulacron.common.result;
 
+import static com.datastax.oss.protocol.internal.ProtocolConstants.ErrorCode.ALREADY_EXISTS;
```

Comment:
can you re-organize your imports so static imports show up at the end?  This is how I have my imports set up (followed format that java driver 4 uses)

![screen shot 2017-08-29 at 11 09 42 am](https://user-images.githubusercontent.com/6889771/29831389-a4792c86-8caa-11e7-9817-b10d9903fa57.png)

Reply:
I'll document this somewhere (wish there was an easier way to enforce it).